### ISSUE TITLE
Clash with org.http4k:http4k-connect-*

### ISSUE THREAD

**Issue by kevin-denver** [jmfayard/refreshVersions#430]
I think the latest version introduced an issue when I am trying to use http4k and http4k-connect in the same project.

https://github.com/jmfayard/refreshVersions/blob/9cde55bca3ccc7eb41d141a12bdbda219f36e0df/plugins/dependencies/src/main/kotlin/dependencies/Http4k.kt#L8

Using `0.11.0` this works:

```
version.org.http4k..http4k-bom=4.12.0.1
version.org.http4k..http4k-connect-bom=3.8.1.0
```

```
api(platform(Libs.http4k_bom))
api(platform(Libs.http4k_connect_bom))
```

Upgrading to `0.20.0` I am getting the following error:

```
 > Could not find org.http4k:http4k-connect-bom:4.12.0.1.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/http4k/http4k-connect-bom/4.12.0.1/http4k-connect-bom-4.12.0.1.pom
```

Notice that it is using http4k's version rather than the one from http4k-connect.

**Comment by LouisCAD** [comment]
Hello, I apologize for taking so long to get back to you. I checked [maven search](https://search.maven.org/search?q=g:org.http4k) to understand the versioning scheme, and I'm adding a version key rule for http4k connect so you'll not get the issue starting next version.

**Comment by LouisCAD** [comment]
If you don't want to wait for the next release, you can use the 0.30.2-SNAPSHOT version that I just published.

**Comment by LouisCAD** [comment]
Fix released in version 0.32.0

I forgot to mention it in the release notes, but it's there.
